libs/core/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.ts

```diff
@@ -66,6 +66,10 @@ export class DynamicPageHeaderComponent
     @Input()
     subtitleWrap = false;
 
+    /** Heading level of the dynamic page title. */
+    @Input()
+    headingLevel = 2;
```

Comment:
You need some type safety for this. The heading level can be a number from 1 to 6. The user should be notified if the number in not one of these.